BASE/src/Microsoft.ApplicationInsights/Properties/AssemblyInfo.cs

```diff
@@ -9,6 +9,7 @@
 [assembly: InternalsVisibleTo("Microsoft.ApplicationInsights.AspNetCore.Tests, PublicKey=" + AssemblyInfo.PublicKey)]
 [assembly: InternalsVisibleTo("Microsoft.ApplicationInsights.WorkerService.Tests, PublicKey=" + AssemblyInfo.PublicKey)]
 [assembly: InternalsVisibleTo("Microsoft.AI.ServerTelemetryChannel, PublicKey=" + AssemblyInfo.PublicKey)]
+[assembly: InternalsVisibleTo("Microsoft.AI.PerfCounterCollector, PublicKey=" + AssemblyInfo.PublicKey)]
```

Comment:
you want to add serviceprofiler/snapshot to this list?

Reply:
I don't have their assembly names nor their public keys. 
i think it's best of those product owners submit a separate PR for this change.